tools/Localizations/Sources/Localizations/ExportTask.swift

```diff
@@ -67,6 +61,30 @@ struct ExportTask {
         try! task.run()
         task.waitUntilExit()
     }
+    
+    // Get Locales that are in the Xcode project
+    private func getProjectLocales() -> (Array<String>) {
+    
+        var localesList:[String] = []
+
+        let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: xcodeProjPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path)
+        let filePaths = blockzillaFolder?.allObjects as! [String]
+        let textFilePaths = filePaths.filter{$0.contains(".lproj")}
+        for txt in textFilePaths {
+ 
+        if let index = txt.firstIndex(of: ".") {
+            let firstPart = txt.prefix(upTo: index)
+            localesList.append(String(firstPart))
+            }
+        }
```

Comment:
Same here, nothing wrong, but if you want to go fancy, you could do this with a `filter()` and `map()` too.